docs/en/how-to-prepare-for-the-transfer-gateway-update.md

```diff
@@ -9,12 +9,9 @@ As part of the Basechain upgrade, we'll be updating the Transfer Gateway. To mak
 
 ## 1. Update the Ethereum Transfer Gateway address
 
-We're updating the transfer gateway smart contracts deployed on the Rinkeby testnet and Ethereum main net. If these are hardcoded in your codebase, make sure you use the addresses listed here:
+We're updating the transfer gateway smart contract deployed on  Ethereum main net. If this is hardcoded in your codebase, make sure you use the address listed on [this page](https://loomx.io/developers/en/mainnet-plasma.html).
```

Comment:
I don't think it makes sense to write
```
If this is hardcoded in your codebase, make sure you use the address listed on [this page](https://loomx.io/developers/en/mainnet-plasma.html).
```
at this point, the address there is still the old one, we haven't deployed the new one yet.